File: DSA/Prime.js
function isPrime(n){
  let isFlag = true
  if(n <= 1){
      isFlag = false
  }
  else{
      for (let i=2; i<=n-1; i++){
          if(n % i == 0){
              isFlag = false
              break
          }
      }
  }
  if (isFlag){
      // console.log(n + " is a prime number")
      return true
  }
  else{
      // console.log(n + " is not a prime number")
      return false
  }
}

// isPrime(5)
function primeInRange(n){
  for (let i=2; i<=n; i++){
     if(isPrime(i)){ 
          process.stdout.write(i + " ")
      
     }
  }
}

primeInRange(50)

File: Python/prime.py

def isprime(num):
  isflag = True
  if num <= 1:
    isflag = False
  else:
    for i in range(2,num):
      if num % i == 0:
        isflag = False
  if (isflag):
    # print("Prime")
    return True
  else:
    # print("Not a Prime")
    return False

def prime(num):
  for i in range(1,num):
    if isprime(i):
      print(i,end=" ,")

prime(50)

